contents/index.tsx

```diff
@@ -2,820 +2,355 @@ import { Speaker } from '@lib/types';
-      </p>
+        <p>He is passionate about Lego, music, devx, communities, management, and people.</p>
+      </div>
     ),
-    twitter: 'borja_burgos',
-    linkedin: 'borjaburgos',
-    github: 'borjaburgos',
+    twitter: 'cirpo',
+    linkedin: 'cirpo',
+    github: 'cirpo',
     abstract: (
-      <>
+      <div>
         <p>
-          <strong>Using Observability to improve the Developer Experience</strong>
+          <strong>Building a DevX team, my story</strong>
         </p>
         <p>
-          Observability is often associated with production and live environments, but it shouldn't
-          be! In this talk we'll explore innovative ways in which modern observability tools and
-          best practices can be leveraged during development to: improve developer productivity,
-          identify regressions earlier in the SDLC, and increase the performance and reliability of
-          our CI/CD workflows.
+          DevX has been an emerging concept for the past two years, and while we are still trying to
+          give it a shared and structured definition in the tech industry, some common questions are
+          already arising, and the most compelling one is: where to start with developer experience?
+          In this presentation, I will talk about my past 4 years of experience setting up a DevX
+          team from scratch, the main challenges, the pain, the gain, and the lessons learned.
         </p>
-      </>
+      </div>
     )
   },
   {
-    name: 'Mike Nikles',
-    company: 'Gitpod',
-    title: 'Senior Developer Success Engineer',
+    name: 'Brendan Falk',
+    company: 'Fig',
+    title: 'Co-founder & CEO',
     image: {
-      url: 'mike-nikles.png'
+      url: 'brendan-falk.png'
     },
-    bio:
-      'Mike Nikles is a software architect who values developer experience, productivity and team morale. For two decades, Mike has taken ideas from inception to production, at startups or as lead for teams at large companies. He works as a Senior Developer & Customer Success Engineer at Gitpod where he helps teams automate their ephemeral development environments.',
-    twitter: 'mikenikles',
-    linkedin: 'mikenikles',
-    github: 'mikenikles',
-    abstract: (
-      <>
-        <p>
-          <strong>Ephemeral Development Environments</strong>
-        </p>
-        <p>
-          Local, long-lived development environments are what we are used to. We spend effort to
-          configure and nurture them, which takes time away from more important tasks. In this talk,
-          Mike and Anton introduce the concept of ephemeral development environments, their benefits
-          and how you can get started right away too.
-        </p>
-      </>
-    )
+    bio: 'Brendan is co-founder / CEO of Fig.',
+    twitter: 'brendanfalk',
+    linkedin: 'brendanfalk',
+    github: 'brendanfalk',
+    abstract: 'Building autocomplete for the terminal '
```

Comment:
I feel like for Brendan Bio & Abstract are much shorter than the other ones

Reply:
Thanks, Julain this is what we got from him in the form. His personal site does not say much as well https://brendanfalk.com/ cc @pawlean Can you suggest something? Thanks

Reply:
Oh I didn't know that. If these are the things he submitted, its fine I guess 🤔